evaluation/mint/run_infer.py

```diff
@@ -0,0 +1,398 @@
+import asyncio
+import functools
+import json
+import logging
+import multiprocessing as mp
+import os
+import pathlib
+import subprocess
+import time
+from concurrent.futures import ProcessPoolExecutor
+from typing import Dict
+
+from datasets import load_dataset
+from datatypes import TaskState
+from env import SimplifiedEnv
+from prompts import ToolPromptTemplate
+from task import ReasoningTask, Task
+from tqdm import tqdm
+
+from evaluation.swe_bench.swe_env_box import DockerSSHBox
+from opendevin.controller.state.state import State
+from opendevin.core.config import config, get_llm_config_arg, get_parser
+from opendevin.core.logger import get_console_handler
+from opendevin.core.logger import opendevin_logger as logger
+from opendevin.core.main import main
+from opendevin.events.action import MessageAction
+
+
+def cleanup():
+    print('Cleaning up child processes...')
+    for process in mp.active_children():
+        print(f'Terminating child process: {process.name}')
+        process.terminate()
+        process.join()
+
+
+def codeact_user_response(state: State, task: Task, task_config: Dict[str, int]):
+    logger.info(f'Gold reference: {task.reference}')
+    logger.info(f'Task config: {task_config}')
+
+    env = SimplifiedEnv(
+        agent_state=state,
+        task=task,
+        task_config=task_config,
+    )
+    result_state: TaskState = env.step()
+
+    logger.info('Result state: ' + result_state.__dict__)
+    logger.info('Latest output: ' + result_state.latest_output)
+
+    msg = result_state.latest_output['content']
+    logger.info('User response:' + msg)
```

Comment:
Hi @xingyaoww, I mean I want to see the `msg` or other variables inside this `codeact_user_response` func for debugging and ensuring the proper response is sent to the agent, but I couldn't find the output of `logger.info` anywhere.

Reply:
You can check the log file. The logger content will show on the log file. When you run the `run_infer.py` file, the cmd will show the location of the log file.

```python
logger.info(f'Writing evaluation output to {output_file}')
```

Reply:
Actually, yesterday when I tried to log something inside `codeact_user_response`, it somehow didn't appear while others can be found. So I asked Xingyao after a long time of debugging. But the issue was just a silly named argument issue which makes the handler not called at all.